### PR TITLE
Feature/navigation drawer

### PR DESCRIPTION
**Pull request by albastardoto**
Mostly changes style values, added custom colors generated using material ui theme generator.

Also added material ui components which I used to add the navigation drawer.

Hopefully  we'll be able to fully switch to using the material ui colors so that we can have as many themes as we want.

### CODE REVIEW

**Review by villemontc — APPROVED** [review]
Great job !

**Review by rombirli — APPROVED** [review]
Good work

**Review by AdrianUlrich — APPROVED** [review]
Very good looking

Can't wait to develop new themes in #90

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,69 +1,71 @@
...
-            </intent-filter>
-        </activity>
-
-        <meta-data
-            android:name="com.google.firebase.ml.vision.DEPENDENCIES"
-            android:value="ocr" />
-
-    </application>
-
+
```
Comment (Sonusiety) [inline app/src/main/AndroidManifest.xml:1]: This white line should be removed so that the build test pass on Cirrus.

**Review by Sonusiety — APPROVED** [review]
Nicely done

**Comment by albastardoto** [comment]
There haven't been any tests as functionality will very likely change and any tests would just be a hindrance in the future

**Pull request merged**